apiExamples/errorGroup.html

```diff
@@ -1,6 +0,0 @@
-<auro-radio-group error="There is an error with this form entry">
```

Comment:
Why don't we need this anymore?

Reply:
We decided we do not want to show support for using a single radio with an error state. I removed the individual example and renamed the `errorGroup.html` to `error.html`. 

If needed, I can add more explanation that validation will only run when we wrap a single radio in a group, like I did for checkbox.